api/src/services/mail/index.ts

```diff
@@ -54,11 +54,7 @@ export class MailService {
 	}
 
 	async send<T>(options: EmailOptions): Promise<T | null> {
-		const payload = await emitter.emitFilter(`email.send`, options, {
-			database: getDatabase(),
-			schema: null,
-			accountability: null,
-		});
+		const payload = await emitter.emitFilter(`email.send`, options, {});
```

Comment:
I think the context should be `null` (or left out all together), otherwise `emitFilter` does not inject the default context, see

https://github.com/directus/directus/blob/122d683eb5a488b4a34ab82e80813cbdc4f0ca95/api/src/emitter.ts#L34
```suggestion
		const payload = await emitter.emitFilter(`email.send`, options);
```

Reply:
Right now the error only seems fixed because context is the empty object. It will return to the error state when letting `emitFilter` create the default context, which is identical to the one passed in previously.